Makefile

```
@@ -5,4 +5,9 @@ build:
 	node_modules/.bin/gulp --cwd ui bundle
 	node_modules/.bin/antora generate $(PLAYBOOK) $(ANTORAFLAGS)
 
+clean:
```

See https://stackoverflow.com/a/2145605
```suggestion
.PHONY: clean
clean:
```